maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -3051,6 +3056,17 @@ private void warnIfIllegalTempDir() throws MojoFailureException
         }
     }
 
+    private void printDefaultSeedIfNecessary()
```

Comment:
This abstract MOJO class is used in the Mockito tests. Can you cover all your changes and extend the existing tests? It would be cool to have tests maybe for larger methods too.

Reply:
I can certainly add a test or two that covers this method. If there are other methods that I should cover or if there are particular tests that I should extend, do let me know.

Reply:
These are the test classes, `AbstractSurefireMojoJava7PlusTest` and `AbstractSurefireMojoTest`, where we add the tests for `AbstractSurefireMojo`.